### DISCUSSION TITLE
How to get TAGGING_SYSTEM to work?

### DISCUSSION THREAD

**Discussion by srinisubramanian** [apache/superset#19931] [Q&A / Help]
I enabled the feature flag TAGGING_SYSTEM in 1.5.0 (after an upgrade from 1.4.0).  Ran superset init and restarted.  However I do not see the tag UI in dashboards or charts.  Any additional steps to be followed?

**Comment by nikhil-kuyya-talentas** [comment]
I don't think tag feature is fully added, there is backend which works but no UI stuff as of to tag the dashboard and charts. Please add your comments i haven't seen much recently if any progress done on this.
